assets/js/modules/analytics-4/components/settings/SettingsForm.stories.js

```diff
@@ -0,0 +1,248 @@
+WithoutModuleAccess.storyName = 'Without module access';
+WithoutModuleAccess.args = {
+	hasModuleAccess: false,
+};
+
+export const PropertyNotAvailable = Template.bind( null );
+PropertyNotAvailable.storyName = 'Property not available';
+PropertyNotAvailable.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).receiveGetAccountSummaries(
+				accountSummaries.map( ( acct ) => ( {
+					...acct,
+					propertySummaries: [],
+				} ) )
+			);
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+
+export const WebDataStreamNotAvailable = Template.bind( null );
+WebDataStreamNotAvailable.storyName = 'Web data stream not available';
+WebDataStreamNotAvailable.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).receiveGetWebDataStreams(
+				{},
+				{
+					propertyID,
+				}
+			);
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+
+export const WithExistingTagMatch = Template.bind( null );
+WithExistingTagMatch.storyName =
+	'With existing tag matching the measurement ID';
+WithExistingTagMatch.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveGetExistingTag( measurementID );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+
+export const WithExistingTagNoMatch = Template.bind( null );
+WithExistingTagNoMatch.storyName =
+	'With existing tag not matching the measurement ID';
+WithExistingTagNoMatch.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveGetExistingTag( 'G-123456789' );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+
+export default {
+	title: 'Modules/Analytics4/Settings/SettingsEdit',
+	decorators: [
+		( Story ) => {
+			const setupRegistry = ( registry ) => {
+				global._googlesitekitDashboardSharingData = {
+					settings: {},
+					roles: [],
+				};
+
+				provideModules( registry, [
+					{
+						slug: 'analytics-4',
+						active: true,
+						connected: true,
+						owner: { login: 'analytics_4-owner-username' },
+					},
+				] );
+
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.receiveGetAccountSummaries( accountSummaries );
+
+				registry.dispatch( MODULES_ANALYTICS_4 ).receiveGetSettings( {
+					accountID,
+					propertyID,
+					webDataStreamID,
+					measurementID,
+					useSnippet: true,
+					canUseSnippet: true,
+					anonymizeIP: true,
+					trackingDisabled: [ 'loggedinUsers' ],
+				} );
+				registry.dispatch( MODULES_ANALYTICS ).receiveGetSettings( {
+					accountID,
+				} );
```

Comment:
Very nitpicky, but can we add a `TODO` comment here so that we can make removing it easier similar to the following?

https://github.com/google/site-kit-wp/blob/74ca48ba71eb8bd91af1a2d957d8d11c0d4f7696/assets/js/modules/analytics-4/components/setup/SetupForm.test.js#L113-L116

Thank you!